.travis.yml

```diff
@@ -21,15 +21,15 @@ cache:
 
 install:
   - cd $TRAVIS_BUILD_DIR/jaxrs-api
-  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+  - mvn install -Pstaging -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
Why do we need this for *Travis*? IIUC this is only needed if we want the CI system to stage the build, but we use *Jenkins* for staging, not *Travis*.

Reply:
Doesn't this add the staging repo we need to fetch our dependencies which are only in staging yet? In this case we will need `-Pstaging`.

Reply:
@spericas Can you please clearly describe *in the PR description* **why** "We need -Pstaging."? Thanks. :-)

Reply:
@mkarg It's in the title of the PR, our final dependencies are in staging until all EE 9 is ready. This is how all EE 9 API are building.

Reply:
You misunderstood the question. I didn't ask why **we** are in staging, but why you think you must add the `-P` flag to the **Travis** build script.  I understand it for **Jenkins**. Travis is not able to put anything into the staging area, nor did it fail without that flag. Travis just checks whether the build fails or not, and it does this pretty well without that flag.